### PR TITLE
Email template added for GFA Verified Upgrade Reminder

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-5201](https://www.youtube.com/watch?v=kw4tT7SCmaY)

@umar-nawaz Please review this. Meanwhile I am testing it on different clients.

### CODE REVIEW

**Comment by umar-nawaz** [comment]
@tasawernawaz LGTM 👍 
Please also send it in email to Me, Simon, Gabe, Patrick and Jeanie

File: templates/upgrade-remider/verified_upgrade_reminder.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+          <!-- Why get verified -->
+          <tr>
+            <td style="background-color: #efefef;">
+              <img src="https://dev.edx.org/sites/default/files/verify-logo1.png" style="float:left; padding-left: inherit;" alt="Verified image">
+              <h4 style="margin:0; padding:8px 10px; float:left; font-weight:normal;">Why Get Verified?</h4>
+            </td>
+          </tr>
+          <tr>
+            <td style="background-color: #efefef; text-align: left;">
+              <ul style="font-size: small; padding-left: 24px; list-style-image: url('https://i.imgsafe.org/aaf28ee432.png');">
```
Comment (umar-nawaz) [inline templates/upgrade-remider/verified_upgrade_reminder.html]: @tasawernawaz "list-style-image:" may not work on all clients. 
 You can verify css properties that work in html emails here: https://www.campaignmonitor.com/css/

**Comment by umar-nawaz** [comment]
ECOM-5201 nice ticket link  😄  @tasawernawaz  thanks. Afreen Afreen ...

**Comment by umar-nawaz** [comment]
header shadow image was used here previously.
https://github.com/umar-nawaz/edx-email-templates/blob/master/templates/news-letter/news-letter-courses.html#L71

**Comment by tasawernawaz** [comment]
Code updated as per your suggestions, Please review. @umar-nawaz

**Comment by umar-nawaz** [comment]
@tasawernawaz thanks lets shoot some emails now. :shipit:

**Comment by umar-nawaz** [comment]
fyi @schenedx

File: templates/upgrade-remider/verified_upgrade_reminder.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+                        <td><img src="https://dev.edx.org/sites/default/files/arrow_head.png" style="margin-right: 5px;" alt=""/></td>
+                        <td>Added motivation to successfully complete the course</td>
+                    </tr>
+                </table>
+              <a style="display:block; outline:none; padding:8px; color:#1e8142;text-align:center; text-decoration: none; border-radius: 3px; border:solid 1px;margin: 10px auto; " href="http://172.16.11.82/edx/email-template/#" target="_blank">
+                Upgrade to a Verified Certificate Today
+              </a>
+            </td>
+          </tr>
+                            <!-- Footer Area -->
```
Comment (schenedx) [inline templates/upgrade-remider/verified_upgrade_reminder.html]: #nit comment mis-alignment

File: templates/upgrade-remider/verified_upgrade_reminder.html [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+          </tr>
+          <tr>
+              <td style="padding: 0;"><img src="https://dev.edx.org/sites/default/files/box-shadow.jpg" width="100%"></td>
+          </tr>
+          <tr>
+            <td style="text-align:center; font-size: 20px;">
+              Upgrade to a Verified Certificate to Earn Credit
+            </td>
+          </tr>
+                  <!-- Course Info  -->
```
Comment (schenedx) [inline templates/upgrade-remider/verified_upgrade_reminder.html]: #nit comment mis-alignment

**Comment by schenedx** [comment]
👍

**Comment by tasawernawaz** [comment]
@schenedx Did you saw email that I sent through https://putsmail.com ? Should I merge it now ?

**Comment by schenedx** [comment]
@tasawernawaz Yes, I saw the email you sent through putsmail. You can certainly merge this PR.
But be aware the marketing people like Patrick and Gabe may have feedbacks during today or tomorrow. So, don't close your ticket until they gives thumbs. You should ping them for review in the ticket comments though

**Pull request merged**